openid-sharedsignals-framework-1_0.md

```diff
@@ -829,6 +829,12 @@ an HTTP POST request to the Configuration Endpoint. On receiving a valid request
 the Event Transmitter responds with a "201 Created" response containing a
 [JSON][RFC7159] representation of the stream’s configuration in the body.
 
+If a stream already exists, and the Transmitter allows multiple streams with the
+same Receiver, the Event Transmitter MUST respond with a new stream ID. If the
+Transmitter does not allow multiple streams with the same Receiver, it MUST respond
+respond with HTTP status code "200 OK" and include a [JSON][RFC7159] representation
```

Comment:
What should happen if the Transmitter does not allow multiple streams, a stream already exists, and the details of that stream are different from the values in this POST request?

Reply:
I think the last sentence should be changed to say

If the Transmitter does not allow multiple streams with the same Receiver, it MUST respond with a "409 - Conflict" error code. The Receiver MAY then GET the existing stream configuration and, if desired, use PATCH or PUT to update or replace the existing stream configuration.

Reply:
From [the description of 409](https://www.rfc-editor.org/rfc/rfc9110#status.409) it looks to be remediable which is not the case here. I think we should stay with 200 OK as return the existing stream config

Adding a new status code would require the existing implementation to change the error handling when this draft is approved. 

Also, may break idempotency expectations with different outcome for same request

Reply:
> Also, may break idempotency expectations with different outcome for same request

Good point! Ok, then we need to address the case where a Receiver tries to "create" a stream a second time with different information. What should happen? An error? A silent update?

Reply:
> What should happen? An error? A silent update?

Another option is to ignore the input and respond with the existing stream. The stream ID would indicate that the transmitter did not create a new stream

Reply:
2023-08-01 call: consensus on 409